dlh_utils/linkage.py

```diff
@@ -435,9 +435,11 @@ def difflib_sequence_matcher(string1, string2):
     +---+---------+----------+----------------+
 
     """
+    
+    if string1 is None or string2 is None:
+      return None
 
-    return SequenceMatcher(
-        a=string1, b=string2) if string1 is not None and string2 is not None else None
+    return SequenceMatcher(a=string1, b=string2).ratio()
```

Comment:
Fine for now but I believe the UDFs will need to be called within the function to be imported from another script. A few other functions have this issue